src/hb-uniscribe.cc

```diff
@@ -603,6 +605,46 @@ hb_uniscribe_font_get_hfont (hb_font_t *font)
   return data ? data->hfont : nullptr;
 }
 
+static hb_blob_t *
+_hb_uniscribe_hfont_reference_table (hb_face_t *face HB_UNUSED, hb_tag_t tag, void *user_data)
+{
+  char *buffer = nullptr;
+  DWORD length = 0;
+
+  HDC hdc = GetDC (nullptr);
+  if (unlikely (!SelectObject (hdc, (HFONT) user_data))) goto fail;
+
+  length = GetFontData (hdc, hb_uint32_swap (tag), 0, buffer, length);
+  if (unlikely (length == GDI_ERROR)) goto fail_with_releasedc;
+
+  buffer = (char *) malloc (length);
+  if (unlikely (!buffer)) goto fail_with_releasedc;
+  length = GetFontData (hdc, hb_uint32_swap (tag), 0, buffer, length);
+  if (unlikely (length == GDI_ERROR)) goto fail_with_releasedc_and_free;
+
+  return hb_blob_create ((const char *) buffer, length, HB_MEMORY_MODE_READONLY, buffer, free);
```

Comment:
You are not releasing the DC if there are no errors.

Reply:
Oh, thanks, done